sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -193,7 +192,6 @@ export class EventHubReceiver extends LinkEntity {
    * Instantiates a receiver that can be used to receive events over an AMQP receiver link in
    * either batching or streaming mode.
    * @hidden
```

Comment:
```suggestion
```

Reply:
Oh.. so we are not being consistently going with `@internal`?

Both `@hidden` and `@internal` would work the same way?

Reply:
I think we want to stick with `@internal` because TypeDoc supports `strip-internal`. `@hidden` isn't _technically_ a tsdoc tag.

Reply:
That said, I don't _think_ we need hidden here since the class is marked _internal_

Reply:
Yeah, @chradek. 
> it removes the @hidden or internal tag on a interface/class member if the interface/class itself already has one of the two tags

Let's just go with `@internal` everywhere?

Reply:
In this PR, I am not taking the call to remove `@hidden` if it is the only tag there. Am only removing it when both `hidden` and `internal` are applied, because that obviously is redundant now